README.md

```diff
@@ -0,0 +1,114 @@
+# html2pdf4doc_python
+
+## Project Overview
+
+html2pdf4doc_python is the Python wrapper/CLI for the HTML2PDF4Doc JavaScript core that drives Chrome/Chromedriver to render HTML exports into PDFs.
+The Python CLI exists to provide a stable automation interface for CI pipelines and batch PDF generation.
+This repository focuses strictly on the Python-side automation layer; the rendering logic remains in the JS core.
+
+## Features
+
+- `print` command runs headless Chrome via Selenium to turn one or more HTML files into PDFs in a single invocation.
+- `get_driver` command pre-downloads and caches a matching Chromedriver, making CI setups reproducible.
```

Comment:
`making CI setups reproducible` is only the second-level reason. The actual reason is to avoid re-downloading chromedriver all the time we print.